firebase-inappmessaging-display/ktx/ktx.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation project(':firebase-components')
     implementation project(':firebase-common:ktx')
     implementation project(':firebase-inappmessaging')
```

Comment:
Maybe we could remove fiam, considering that fiam-ktx already has a dependency on it?